tests/notebook-controller-overlays-application_test.go

```diff
@@ -49,7 +49,7 @@ spec:
      - jupyter
      - notebook
      - notebook-controller
-     - jupyterhub
+     - jupyterhub  
```

Comment:
Why do we still have jupyterhub here? That package no longer exists.

Reply:
This is why: https://github.com/kubeflow/manifests/blob/master/jupyter/notebook-controller/overlays/application/application.yaml#L35

@jlewi @kkasravi I real don't want to be in the same situation than we you asked to remove Tiller and Heritage at the same time people were reverting changes to go back to deprecated version of Ingress, boggus changes which are still here by way as as the Tiller stuff.

So tell me what you want....I can drop the "generation code" commit from this PR so that you can merge...and one of you will have to run make generate again later

Reply:
it is unrelated to what you're trying to get merged. 
It's ok with me if we merge this and then remove jupyter in a subsequent PR.
@jlewi ?

/lgtm